phpBB/posting.php

```diff
@@ -1388,8 +1386,6 @@
 				'post_author_name',
 				'update_message',
 				'update_subject',
-				'submit',
-				'error',
```

Comment:
@nickvergessen The error variable only has the mentioned effect prior to the if statement this event is in. In here, this doesn't have any effect.